src/infrasys/time_series_manager.py

```diff
@@ -321,7 +321,7 @@ def remove(
             self._storage.remove_time_series(
                 time_series[uuid], connection=_get_data_connection(connection)
             )
-            logger.info("Removed time series {}.{}", time_series_type, variable_name)
+            logger.info("Removed time series {}", uuid)
```

Comment:
Can we include all the info? I suggest to add back the time_series_type and variable_name since I think it does not provide a good info message (unless you remember the uuids). Also the signature of this function requires to pass variable name.

Reply:
It's not so straightforward. First, variable_name and time_series_type are optional, so any log message has to handle that. Second, this log message only runs when there are no more references to the time array - so you don't always see a message. Third, if variable name is not specified, more than one variable name could be deleted.

I would say that I corrected the message to be accurate in all cases. I will acknowledge that it isn't super helpful. All you really see is how many arrays were actually deleted. If we wanted to do more work to log information about metadata deletions as well as data deletions, we could do that. That is more work than I have time for at the moment.